apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -589,5 +589,41 @@ public Response dexEthInfo(@Context SecurityContext securityContext) throws NotF
             return Response.ok().build();
         }
     }
+    
+    
+    
+    @GET
+    @Path("/flush")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(tags = {"dex"}, summary = "Flush temporary keys", description = "cleanup after the exchange routine")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Exchange offers"),
+            @ApiResponse(responseCode = "200", description = "Unexpected error") })
+    public Response flush(      @Parameter(description = "User account id.") @QueryParam("accountid") String accountIdStr,                               
+                                @Context HttpServletRequest req) throws NotFoundException {
+
+        log.debug("flush: accountIdStr: {}", accountIdStr);
+        Long accountId = null;
+        try {
+            if(!StringUtils.isBlank(accountIdStr)){
+                accountId = Long.parseUnsignedLong(accountIdStr);
+            }
+            String xpassphrase = Convert.emptyToNull(ParameterParser.getPassphrase(req, true));
+            if(xpassphrase == null) {
+                log.error("null passphrase is unacceptable");
+                return Response.status(Response.Status.OK).entity(JSON.toString(incorrect("passphrase", "Can't be null."))).build();
+            }
+            if ( service.flushSecureStorage(accountId, xpassphrase) ) {
+               return Response.ok().build(); 
```

Comment:
Maybe better to return something?

Reply:
Maybe yes, maybe no, maybe rain, maybe snow. I thought about that in the very beginning, but... It would be better not to indicate user with something except for OK in this case, in order not to alarm the public. Thus, "200 OK" would be enough in this case.

Reply:
p.s. My idea was confirmed by the management.